### ISSUE TITLE
Site forces logout when attempting to tag a character that is not present in a location.

### ISSUE THREAD

**Issue by LawOfTheLand** [RolePlayGateway/roleplaygateway#195]
Have tested and confirmed that this happens in not just The Multiverse, but also in Test Roleplay: This Title Is Win.

**Comment by lostamongtrees** [comment]
Can't seem to replicate, are there any circumstances specific? maybe more than one tab open, etc?

**Comment by Lady-Awinita** [comment]
> Can't seem to replicate, are there any circumstances specific? maybe more than one tab open, etc?

Yes I can give an example.
I posted after 4 tries to do so, as Blackmane in Silver Palace Throne Room, once the post was made via Place Page directly and Blackmane clicked on to tag her in the post, I hit ADD POST, instead of getting to the screen of "You have posted to this roleplay" I am instead redirected to the Login Screen.

The only other tab I have open is RPG Chat

**Comment by lostamongtrees** [comment]
Something odd happend
I tagged a character that was not present at the location
I did not post
I refreshed the page
I received a notification that the character had been written about.
![image](https://user-images.githubusercontent.com/52435234/71050720-f4993d00-2113-11ea-9d7f-d8aa72a43039.png)

I was able to replicate the issue
![image](https://user-images.githubusercontent.com/52435234/71050777-24e0db80-2114-11ea-811a-41e5ba2e1520.png)

**Comment by lostamongtrees** [comment]
They also show up down here in the recently tagged
![image](https://user-images.githubusercontent.com/52435234/71050809-3cb85f80-2114-11ea-8b9b-d959aa5afadb.png)

**Comment by martindale** [comment]
> Something odd happend
> I tagged a character that was not present at the location
> I did not post
> I refreshed the page
> I received a notification that the character had been written about.
> ![image](https://user-images.githubusercontent.com/52435234/71050720-f4993d00-2113-11ea-9d7f-d8aa72a43039.png)
> 
> I was able to replicate the issue
> ![image](https://user-images.githubusercontent.com/52435234/71050777-24e0db80-2114-11ea-811a-41e5ba2e1520.png)

I think this is because you are either 1) owner of the location, or 2) because you are a GM over the universe in question.  Can you check that this is the case?

Were you logged out when this happened?

**Comment by lostamongtrees** [comment]
![image](https://user-images.githubusercontent.com/52435234/71143661-64c5c280-21e9-11ea-8b20-1cc20390446d.png)

it seems tagging in general provokes a log out

**Comment by Lady-Awinita** [comment]
> ![image](https://user-images.githubusercontent.com/52435234/71143661-64c5c280-21e9-11ea-8b20-1cc20390446d.png)
> 
> it seems tagging in general provokes a log out

Confirmed as I was tagging 2 characters phyically present when attempting to make a post, I got logged out, but ont he second attempt the post went through.

**Comment by lostamongtrees** [comment]
Cannot replicate issue anymore & closing!
Thank you again @Lady-Awinita & @LawOfTheLand for catching this!

**Comment by Lady-Awinita** [comment]
I only managed to find a temporary workaround so far; Manually typing in the character names seems to avoid getting logged out when making posts
